src/config.cpp

```diff
@@ -92,6 +92,67 @@ bool Config::validate() {
             std::cerr << "Error: --task parameter not set." << std::endl;
             return false;
         }
+#if (PYTHON_DISABLE == 1)
```

Comment:
#if (PYTHON_DISABLE == 1) can be removed IMHO. We base optimum-cli functionality on external tools/python/environment not on internal ovms compilation flags. This check should be generic until I change it in optimum-cli enabling PR. @dtrawins

Reply:
I think this is a valid check. The version without python shouldn't pull models outside of OV organization. Otherwise the pulled model will not work. It might change in the future but for how it looks reasonable.
@rasapala  do you have in mind replacing it with a logic for checking if optimum-cli is present in runtime? It would be also fine when we have this detection.